doc/sphinx/language/core/modules.rst

```diff
@@ -1010,3 +1010,66 @@ subdirectories of path). See the command :cmd:`Declare ML Module` in
 
 See :ref:`command-line-options` for a more general view over the Coq command
 line options.
+
+.. _controlling-locality-of-commands:
+
+Controlling the scope of commands with locality attributes
+----------------------------------------------------------
+
+Many commands produce an effect that will persist in a given scope,
+which can typically be the one of the section or of the module in
+which the command was called. Locality :term:`attributes <attribute>`
+can be used to change the scope of this effect. Note that some
+commands, like :cmd:`Arguments` do not have the same default locality
+when they are used inside or outside sections.
+
+.. attr:: local
+
+   In most cases, the :attr:`local` attribute limits the effect of the
+   command to the current scope (section or module).
+
+   The only exception is when applied to :cmd:`Definition`,
+   :cmd:`Theorem` or variants. For these commands, the effect of the
+   :attr:`local` attribute will be that the defined object will not be
+   available through a shorter name when importing the module in which
+   it is defined.
+
+   There is an alternative syntax for the :attr:`local` attribute
+   using the ``Local`` prefix (see :n:`@legacy_attr`).
+
+   .. note::
+
+      For some commands, this is the only supported locality within
+      sections (e.g., for :cmd:`Notation`, :cmd:`Ltac` and :ref:`Hint
+      <creating_hints>` commands).
+
+.. attr:: export
+
+   The :attr:`export` attribute makes the effect of the command
+   persist when the section is closed and when the module containing
+   the command is imported.
+
+   This attribute is supported in particular by the :ref:`Hint
+   <creating_hints>`, :cmd:`Set` and :cmd:`Unset` commands, although
+   for :ref:`Hint <creating_hints>` commands, it is not supported
+   inside sections.
```

Comment:
```suggestion
   This attribute is supported in particular by the :ref:`Hint
   <creating_hints>`, :cmd:`Set` and :cmd:`Unset` commands, although
   for :ref:`Hint <creating_hints>` commands, it is not supported
   inside sections.
```

is "in particular" needed?  There are others that are not important.

Reply:
The goal was to prevent the documentation to become incorrect as the list of commands supporting the attribute would expand.

Reply:
You could say "Commands supporting this attribute include ...".  "In particular" implies that the listed items are somehow more important that those that are not listed.